### ISSUE TITLE
egs_internal_source error 2 on congfigure

### ISSUE THREAD

**Issue by Mighty-Raindeer** [clrp-code/egs_brachy#24]
on initial configuration, egs_internal_source is not compiling into c++ library. Shows error 2.
[egs_internal_source error 2.txt](https://github.com/clrp-code/egs_brachy/files/8036113/egs_internal_source.error.2.txt)

**Comment by MartinMartinov** [comment]
Good catch, the ifstream input didn't work for the same generation of compilers as the rest of the code.  It has now been addressed in this commit:

https://github.com/clrp-code/EGSnrc_CLRP/commit/3f98fe581750605892967d6b811c12e034f61c94

you can pull the EGSnrc_CLRP repository again to grab the change.  Hope it works this time, and thanks for catching the bug.
